src/Sarif.FunctionalTests/v2/ObsoleteFormats/ComprehensivePrereleaseTwoZeroZero.sarif

```diff
@@ -22,6 +22,17 @@
           "copyright": "Copyright (c) 2016 by Example Corporation. All rights reserved."
         }
       },
+      "conversion": {
+        "tool": {
+          "name": "CodeScanner Legacy File Format Converter",
+          "fullName": "CodeScanner Converter Tool",
+          "version": "4.3",
+          "semanticVersion": "4.3.0",
+          "fileVersion": "4.3.0.0",
```

Comment:
>fileVersion [](start = 11, length = 11)

NOTE: this file and many other test files are left in the old pre-release format by design to ensure that the prerelease transformation layer continues to function properly. Once the design ships, we will make a comprehensive pass to 'upsell' everything to final SARIF v2 (except for the obsolete example).